pallets/simple-staking/src/benchmarking.rs

```diff
@@ -3,7 +3,7 @@
 use super::*;
 
 #[allow(unused)]
-use crate::Module as Template;
+use crate::Module as simple_staking;
```

Comment:
```suggestion
use crate::Module as SimpleStaking;
```

`Types` should CamelCase. 

Any why are you not using `Pallet`?